src/repository/screens/repository-file.screen.js

```diff
@@ -105,17 +113,29 @@ class RepositoryFile extends Component {
                 </Text>
               </View>
 
-              {!this.isImage(fileType) &&
+              {!this.isImage(fileType) && !this.isKnownType(fileType) &&
```

Comment:
What about creating a constant on above `return` that goes something like: `const isKnownImage = !this.isImage(fileType) && !this.isKnownType(fileType);` so it's clearer for new comers to the repo what does that 2 conditions mean?

Besides that, looks good to me! :shipit: